Docs/developer_guide/extensions.md

```diff
@@ -227,8 +225,55 @@ After installing an extension, the directories are added to revision-specific se
 - Folders containing modules bundled within an extension are added to Modules / AdditionalPaths. This ensures that libraries associated with modules are found.
 - Folders containing third-party dynamic libraries, Python libraries, etc. are added to LibraryPaths, Paths, PYTHONPATH, QT_PLUGIN_PATH. This ensures that libraries associated with modules can be successfully loaded.
 
+(extension-catalog-entry-file)=
+## Extension catalog entry file
+
+An extension catalog entry file is a JSON file describing how to build and package a Slicer extension.
+
+It is named after the extension name (e.g `ExtensionName.json`) and may include the following entries:
+
+
+```{list-table}
+:header-rows: 1
+
+* - Name
+  - Description
+  - Required
+* - category
+  - Extension category.
+  - Y
+* - scmurl
+  - Read-only url used to checkout the extension source code.
+  - Y
+* - scmrevision
+  - Revision allowing to checkout the expected source code.
+  - N
+* - scm
+  - Type of revision control system. Default to `git`.
+  - N
+* - build_dependencies
+  - List of extensions required to build this extension. For example: `["extensionA", "extensionB"]`.
+  - N
+* - build_subdirectory
+  - Name of the inner build directory in case of superbuild based extension. Default to `.`.
+  - N
+* - enabled
+  - Specify if the extension should be enabled after its installation. Valid values are `true` (default) = enabled; `false` = disabled.
+  - N
+```
+
+The full schema for the extension catalog entry file is below:
+
+```{jsonschema} ../../Schema/slicer-extension-catalog-entry-schema-v1.0.0.json
```

Comment:
Looks like I missed something ... the table is not generated, see https://slicer--7629.org.readthedocs.build/en/7629/developer_guide/extensions.html#extension-catalog-entry-file

Reply:
I was missing the entry in the extension list in `conf.py`